src/main/scala/dpla/ingestion3/entries/ingest/IngestRemap.scala

```diff
@@ -51,6 +53,13 @@ object IngestRemap extends MappingExecutor
       .set("spark.kryoserializer.buffer.max", "200")
       .setMaster(sparkMaster)
 
+
+    Utils.deleteRecursively(new File(harvestDataOut))
```

Comment:
@mdellabitta I don't think we want to do this here ;)

Reply:
:man_facepalming:

Reply:
I was just being thorough!